YeusofGeekyTrail.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1 id="title">Yeusof & Geeky Trail</h1>
+    <p id="description">This is our our code</p>
+    <form id="survey-form" action="">
```

Comment:
- [ ] Please try to be more professional in your comments and add solid reasons why `action` and `method` both attributes are important and what is the significance of both. Kindly try to explain to the student as a good code reviewer so that the student will find it easy to implement the things without confusion and be more polite.

## Your comment

>  Give or specify the method and action to the form to make it clear and formal.